src/components/tools/geometry-tool/geometry-content.tsx

```diff
@@ -415,6 +436,41 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
     }
   }
 
+  private closeCommentDialog = () => {
+    this.setState({ selectedComment: undefined });
+  }
+
+  // TODO: Create comments after the dialog is complete + prevent empty comments
```

Comment:
Well, you can still create empty comments, mostly because it was easier, not because it was an explicit design decision. I figured I'd leave this note to help make that clear, maybe I'd be better off with a PT story.